canopy/canopyjs/src/components/navigation/SideNav.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2018-2020 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import classnames from 'classnames';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { CanopyContext } from '../../CanopyContext';
```

Comment:
Instead of this, you can just `import { useUserSaplings } from '../../CanopyContext'` and then in the component `const userSaplings = useUserSaplings()`. This way you can remove that `CanopyContext.Consumer` wrapper.

Reply:
updated